ci_framework/roles/build_containers/tasks/hotfix.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- name: Run tcib hotfix command
+  ansible.builtin.command: >
```

Comment:
care to use the following syntax:
```YAML
- name: Run tcib hotfix command
  ansible.builtin.command:
    cmd: >-
      openstack ...
```

It's converging with all of the other calls.

Reply:
done